src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoCalendarWeekRule.cs

```diff
@@ -45,33 +45,18 @@ public void TestInvalidValue()
             });
         }
 
-        // TestLocaleFirstDay: Verify value of property CalendarWeekRule for specific locale
-        [Fact]
-        public void TestLocaleFirstDay()
-        {
-            CultureInfo myTestCulture = new CultureInfo("en-US");
-            DateTimeFormatInfo dti = myTestCulture.DateTimeFormat;
-            CalendarWeekRule actual = dti.CalendarWeekRule;
-            Assert.Equal(CalendarWeekRule.FirstDay, actual);
-        }
-
-        // TestLocaleFirstFourDayWeek: Verify value of property CalendarWeekRule for specific locale
-        [Fact]
-        public void TestLocaleFirstFourDayWeek()
+        // Verify value of property CalendarWeekRule for specific locales
+        [Theory]
+        [InlineData("en-US")]
+        [InlineData("br-FR")]
+        public void TestLocale(string localeName)
```

Comment:
While I appreciate the complexities of br-FR's calendar rule, I'm wondering if we couldn't help clarity by having some more fixed points called out in something like

```C#
[Theory]
[InlineData("en-US", CalendarWeekRule.FirstDay)]
[InlineData("fr-FR", CalendarWeekRule.SomeConst)]
[InlineData("ja-JP", CalendarWeekRule.SomeConst)]
public void VerifyCalendarWeekRule(string localeName, CalendarWeekRule expected)
{
...
```

just so that we can get breadth-at-a-glance.

Then have br-FR and maybe a couple other complex cases in another test method group.